pkg/users/git.go

```diff
@@ -71,6 +73,14 @@ func (r *GitUserResolver) Resolve(user *gits.GitUser) (*jenkinsv1.User, error) {
 		}
 		new := r.GitUserToUser(gitUser)
 		id = gitUser.Login
+		// Check if the user id is available, if not append "-<n>" where <n> is some integer
+		for i := 0; true; i++ {
+			_, err := r.JXClient.JenkinsV1().Users(r.Namespace).Get(id, v1.GetOptions{})
+			if err == nil {
```

Comment:
Shouldn't this be negated? We break out if we couldn't retrieve the user...

Better still is to use `errors.IsNotFound`:
https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go#L457

Reply:
Thanks, good point